packages/go_router/doc/named-routes.md

```diff
@@ -0,0 +1,46 @@
+Instead of navigating to a route based on the URL, a GoRoute can be given a unique
```

Comment:
What's the benefit of having a separate page for named routes? Would it be better to move this content to sections in the navigation and redirection topics? Or could we at least link to the named routes topic from the navigation and redirection topics?

Reply:
I would prefer to keep them separate - named routes are opt-in, you don't need to use them to get up and running.

Reply:
I can add some links for now - let me know if you disagree.